src/pybind/mgr/cephadm/serve.py

```diff
@@ -112,6 +114,39 @@ def serve(self) -> None:
             self.log.debug("serve loop wake")
         self.log.debug("serve exit")
 
+    def _check_certificates(self) -> None:
+        for d in self.mgr.cache.get_daemons_by_type('grafana'):
+            cert = self.mgr.get_store(f'{d.hostname}/grafana_crt')
+            key = self.mgr.get_store(f'{d.hostname}/grafana_key')
+            if (not cert or not cert.strip()) and (not key or not key.strip()):
+                # certificate/key are empty... nothing to check
+                return
+
+            try:
+                get_cert_issuer_info(cert)
+                verify_tls(cert, key)
+                self.mgr.remove_health_warning('CEPHADM_CERT_ERROR')
```

Comment:
can we rename this error the `CEPHADM_GRAFANA_CERT_ERROR`?

Reply:
forget this one, just read through the other comments and got the context that this is meant to be expanded foe other certs we manage.